resources/delete-plugin.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+num_min_required_params=1
+num_max_required_params=2
+num_params=$#
+
+function print_usage() {
+    echo "usage: delete-plugin[.sh] <plugin-name> --force"
+    echo "1st parameter: name of the plugin"
+    echo "2nd parameter: '--force' flag to start the actual execution of the deletion process"
+    echo
+    echo "Deletes the plugin passed as 1st parameter."
+    echo
+    print_backup_info
+    echo
+    echo "To execute the deletion of the plugin add --force flag as 2nd parameter"
+}
+
+function print_backup_info() {
+    echo "As the removal of the plugin may also result in changes to the database,"
+    echo "it is recommended to make a backup of the database before removing the plugin."
+}
+
+if [[ $# -lt $num_min_required_params  || $# -gt $num_max_required_params ]]; then
+    echo "Wrong number of arguments - Plugin name must be given as the first argument."
+    echo
+    print_usage
+    exit 1
+fi
+
+plugin_name=$1
+
+if [[ $num_params -eq 1 ]]; then
+    print_backup_info
+    echo
+    echo "Insert the flag --force at the end of the command to definitely uninstall the selected plugin $plugin_name"
+    echo
+    print_usage
+    exit 1
+fi
+
+
+force_param=$2
+if [[ $force_param == "--force" ]]; then
+  # Automate uninstall steps from official Redmine guide -
```

Comment:
When I mistype a plugin I want to remove from Redmine I see a lot of output. Only the last couple of lines tell me that there was actually a problem. While this is technically works out good enough I'd like rather a fail-early approach here so that inattentive admins don't regard the surprisingly large output as success. Like `Could not remove redmine plugin ' testtesttest' because it does not seem to be installed`.

Example output:
```
cesapp command redmine delete-plugin asdf --force
Delete plugin asdf
Creating scope :system. Overwriting existing method Enumeration.system.
Creating scope :sorted. Overwriting existing method Group.sorted.
Creating scope :sorted. Overwriting existing method User.sorted.
Creating scope :having_mail. Overwriting existing method User.having_mail.
Creating scope :visible. Overwriting existing method AgileQuery.visible.
Creating scope :visible. Overwriting existing method AgileVersionsQuery.visible.
DEPRECATION WARNING: Using a dynamic :action segment in a route is deprecated and will be removed in Rails 6.0. (called from instance_eval at /usr/share/webapps/redmine/config/routes.rb:395)
DEPRECATION WARNING: Using a dynamic :action segment in a route is deprecated and will be removed in Rails 6.0. (called from instance_eval at /usr/share/webapps/redmine/config/routes.rb:395)
DEPRECATION WARNING: Using a dynamic :action segment in a route is deprecated and will be removed in Rails 6.0. (called from instance_eval at /usr/share/webapps/redmine/config/routes.rb:395)
DEPRECATION WARNING: Using a dynamic :action segment in a route is deprecated and will be removed in Rails 6.0. (called from instance_eval at /usr/share/webapps/redmine/config/routes.rb:395)
DEPRECATION WARNING: Using a dynamic :action segment in a route is deprecated and will be removed in Rails 6.0. (called from instance_eval at /usr/share/webapps/redmine/config/routes.rb:395)
DEPRECATION WARNING: Using a dynamic :action segment in a route is deprecated and will be removed in Rails 6.0. (called from instance_eval at /usr/share/webapps/redmine/config/routes.rb:395)
DEPRECATION WARNING: Using a dynamic :action segment in a route is deprecated and will be removed in Rails 6.0. (called from instance_eval at /usr/share/webapps/redmine/config/routes.rb:395)
Plugin asdf was not found.
2022-01-17T13:56:43+01:00 [ERRO] cesapp.go:checkMainError:141 command delete-plugin failed on container redmine, with exit code 1
```